tensorflow/lite/kernels/resize_bilinear_test.cc

```diff
@@ -68,6 +68,10 @@ class ResizeBilinearOpModel : public SingleOpModel {
     PopulateTensor(input_, data);
   }
 
+  TfLiteTensor* GetInputTensor(int index) { return interpreter_->tensor(input_); }
```

Comment:
Why are you not using the existing methods? See how the other test cases are seeing up the input and checking the output